src/libraries/System.Net.Security/tests/FunctionalTests/SslStreamRemoteExecutorTests.cs

```diff
@@ -34,6 +34,13 @@ public async Task SslKeyLogFile_IsCreatedAndFilled(bool enabledBySwitch)
                 return;
             }
 
+            if (PlatformDetection.IsOpenSslSupported && PlatformDetection.OpenSslVersion >= new Version(3, 5, 0))
```

Comment:
Consider 

```suggestion
            if (PlatformDetection.IsOpenSsl3_5)
```

Reply:
I though IsOpenSsl3_5 test exact version, it is named little bit confusing. Changed.